modules/globebrowsing/tile/chunktile.h

```diff
@@ -34,6 +34,12 @@ namespace openspace {
 namespace globebrowsing {
 
 struct ChunkTile {
+	ChunkTile() : tile(Tile::TileUnavailable) {};
```

Comment:
Actually... It does compile on mac but not on windows. I think it is because Tile is a member of ChunkTile and Tile is not an aggregate type.